brion/morphology.h

```diff
@@ -81,6 +86,14 @@ class Morphology : public boost::noncopyable
      */
     BRION_API Vector2isPtr readApicals() const;
 
+    /**
+     * @return perimeters of the cross sections for each point of the morphology
+     *         in micrometers.
+     * @throw std::runtime_error if empty for FAMILY_GLIA
```

Comment:
Hmm, good point. Ideally we should check what we can check in the constructor. Will add it, at least for version 1 and 1.1.